content/en/admin/troubleshooting.md

```diff
@@ -30,3 +30,13 @@ Check that you are specifying the correct environment with `RAILS_ENV=production
 ## **I encountered a compilation error while executing `RAILS_ENV=production bundle exec rails assets:precompile`, but no more information is given. How to fix it?**
 
 Usually it's because your server ran out of memory while compiling assets. Use a swapfile or increase the swap space to increase the memory capacity. Run `RAILS_ENV=production bundle exec rake tmp:cache:clear` to clear cache, then execute `RAILS_ENV=production bundle exec rails assets:precompile` to compile again. Make sure you clear the cache after a compilation error, or it will show "Everything's OK" but leave the assets unchanged.
+
+## **I am getting this error: `Read-only file system @ dir_s_mkdir`. Why?**
+
+Currently mastodon uses hardening sandbox mode implemented by SystemD. More details with the sandboxing options to get acquainted here - https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Sandboxing.
```

Comment:
I would suggest removing this paragraph, since all of the inflammation the user needs is repeated more clearly in the following paragraph.